.yarnrc.yml

```diff
@@ -0,0 +1,10 @@
+nodeLinker: node-modules
+
+npmAuthToken: "${NPM_TOKEN}"
```

Comment:
The [GitLab docs](https://docs.gitlab.com/ee/user/packages/yarn_repository/#publish-configuration) mention using

```yaml
npmScopes:
  <my-org>:
    npmPublishRegistry: 'https://<your_domain>/api/v4/projects/<your_project_id>/packages/npm/'
    npmAlwaysAuth: true
    npmAuthToken: '<your_token>'
```

Leaving this indented to `npmScopes` will cause `yarn npm publish` to fail due to authentication errors, so it needs to be at the top-level instead.